spec/spec_helper.rb

```diff
@@ -0,0 +1,116 @@
+# Generated by rspec-sinatra. (2015-11-17 11:08:51 +0000)
+
+
+require 'coveralls'
+require 'simplecov'
+
+SimpleCov.formatters = [
+  SimpleCov::Formatter::HTMLFormatter,
+  Coveralls::SimpleCov::Formatter
+]
+Coveralls.wear!
+
+require 'byebug'
+require 'capybara'
+require 'capybara/rspec'
+require 'rspec'
+
+
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'app.rb')
+
+
+Capybara.app = TillTest
+# hmm - should be generated by `rspec --init` directly rather than copy paste ...
+
+# This file was generated by the `rspec --init` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
+# The generated `.rspec` file contains `--require spec_helper` which will cause this
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)